src/app/views/query-runner/util/iframe-message-parser.ts

```diff
@@ -0,0 +1,111 @@
+function isVerb (word: string): boolean {
```

Comment:
Global to PR:
Document the functions, [provide copyright headers](https://docs.opensource.microsoft.com/releasing/copyright-headers.html)